packages/x-charts/src/internals/stacking/offset/offsetDiverging.ts

```diff
@@ -0,0 +1,43 @@
+/**
+ * Adapted from D3.js's offsetDiverging function https://github.com/d3/d3-shape/blob/main/src/offset/diverging.js
+ * The original function return 0 values stacked at 0, this implementation put them either
+ * at the positive or negative sum according to the original data.
```

Comment:
This isn't very clear to me. What do you mean by "according to the original data"?

If I have a stack with the values 10, and 0, will the returned range be [0, 10], [10, 10] or [0, 10], [0, 0]?

From my understanding of this sentence, I'd expect the result to be the former, but according to the tests it's the latter.